src/wfcomman.c

```diff
@@ -961,7 +961,7 @@ AppCommandProc(register DWORD id)
 			   }
 			   lstrcat(szToRun, TEXT("\\ConEmu\\ConEmu64.exe"));
 			   if (PathFileExists(szToRun)) {
-				   wsprintf(szParams, TEXT(" -Single -Dir \"%s\""), szDir);
+				   wnsprintf(szParams, MAXPATHLEN + 20, TEXT(" -Single -Dir \"%s\""), szDir);

```

Comment:
maybe use `sizeof(szParams) / sizeof(szParams[0])` rather than hardcoding buffer size, which might become desynced over time

Reply:
Fixed. Thanks for the suggestion!